script/Ciel/Day 6/06_01_CIEL06_1_2.txt

```diff
@@ -236,73 +249,80 @@ Honestly, I'm doubting my own explanation too.
 [sha:e1bae1f781e9b6a0419d5a8a3b64820a97bc93a3]{
 -- Page 80, Offset 13365.
 -- 秋葉は正面から見据えてくる。
-Akiha is looking directly at me.
+Akiha is staring me down.
 }
 [sha:7d504228f4648107d4fd4ce1ce6ff6e52db535a3]{
 -- Page 80, Offset 13366.
 -- “絶対に誤魔化されませんから”という意思がこれでもかと伝わってくる、末恐ろしい眼力だった。
-I could ominously perceive how strong her refusal to be deceived was.
+Her gaze is absolutely terrifying. It's crystal clear that she will not let herself be fooled.
 }
 [sha:72c5940b0f5f3c77dc4b829f421fc6291b6a63e4]{
 -- Page 81, Offset 13367.
 -- 「いや、どのような人物、と言われても、一言では説明できないというか……」
-"Ah, 'what kind of person' you ask me, but that's not something I can explain in a few words..."
+"Weeell, even if you ask me what kind of person they are, that's not the sort of thing I can sum up in one word..."
 }
 [sha:e732f934d891129eaddb5db6c78b344fe04609d9]{
 -- Page 81, Offset 13368.
 -- というか<人|・>物なのか、アイツ。
-// There was an emphasis on <人|・>物, but there's probably no way to reflect that.
-Anyway, can that woman even be considered a person?
+I mean, for starters, does that woman even %{i}count%{/i} as a person?
 }
 [sha:9d7f32e440855036d7c4474876d19058607375ba]{
 -- Page 82, Offset 13369.
 -- 「そう。言えない、という事は後ろめたい事がある、という事よ、兄さん」
-"I see. You won't tell. Meaning there must be something you feel guilty about, Nii-san."
+// Very nearly offset 14166
+"I see. If you cannot tell me, then there must be something you feel guilty about, Nii-san."
 }
 [sha:578770bb924e534b815acf7d690df6a60c273e6a]{
 -- Page 82, Offset 13370.
 -- 氷のような視線が突き刺さる。
-Her ice cold stare is piercing.
+Her ice cold gaze bores into me.
 }
 [sha:8fa2239ff985f69f0db80f4eee76a62256c1930c]{
 -- Page 82, Offset 13371.
 -- 目を逸らすと余計追い詰められそうなので、負けじと秋葉の顔を見つめる。
-If I look away, I'll be pushed even further into a corner, so I undauntedly stare back at Akiha.
+I feel like I'll only pushed even further into a corner if I look away,
+ so I resolutely stare back at her.
 }
 [sha:60963eda62a94abf1fb136e8d90f3cc7451c3104]{
 -- Page 83, Offset 13372.
 -- ……………いや、まったく関係ないけど。
-This is completely unrelated, but...
+...Not to go off on a tangent, but...
 }
 [sha:919212ce9e29adbd43ee82fb78a5727635037f61]{
 -- Page 83, Offset 13373.
 -- こう改めて見ると、秋葉には昔の面影がない。
-Looking at her once more, she has completely changed from when we were young.
+// Basically offset 14170
+Looking at her with fresh eyes like this, I come to realize how much Akiha has really changed.
 }
 [sha:52172707bcebfba487f1cd689a276b8367841750]{
 -- Page 83, Offset 13374.
 -- 凛と伸びた背筋、はっきりとした口調、ゆらぐ事のない瞳。
-She is standing straight, in a dignified manner, her words are clear, and her gaze unwavering.
+// Excerpt of 14172
```

Comment:
hm, i wonder if a manual lint for these might be a good idea. Something like //@similar{14170} so that changes to those lines log a warning.

Reply:
I did add `LintConsistency:offset`, but that checks strict equivalency - doing something fuzzier would be a bit harder since you'd need to guesstimate what parts are just different between lines, and what is changed within the common section

Reply:
right, that's why i suggested it would just log a warning. If you modify the line it'll just say "hey, did you check the other line?"

Reply:
Yeah, it would just have to be stateful / involve git somehow. I'll ponder whether there's an easy way to implement something like that.